hyrax.gemspec

```diff
@@ -26,6 +26,12 @@ EOF
   spec.version       = Hyrax::VERSION
   spec.license       = 'Apache2'
 
+  # Be pessimistic, rails does not follow sem-ver conventions and
+  # minor version releases can include breaking changes; see
+  # http://guides.rubyonrails.org/maintenance_policy.html
+  # https://github.com/projecthydra-labs/hyrax/issues/1028
+  spec.add_dependency 'rails', '>= 5.0.0', '< 5.2'
```

Comment:
Obviously, I coded and prefer the pessimistic constraint to avoid potential breaking changes downstream.  The safe upgrade process is to add a new minor release version to travis when it becomes available, read the release notes (esp. any deprecation warnings etc.) and see it the specs pass.  If or when the specs pass, bump up the pessimistic constraint to allow the new minor version.